src/utils.js

```diff
@@ -119,55 +119,108 @@ module.exports = class Utils {
   static selectFiles(config = {}) {
     return new Promise((resolve, reject) => {
       /**
-       * Create a new INPUT element
-       * @type {HTMLElement}
+       * Check
```

Comment:
the last one. check what?

Reply:
Sorry, I'm late.
I edited the comments, would it be okay like this?